File: Php/app/Urls/CashgameListingUrlModel.php
namespace app\Urls{

	use app\RouteFormats;
	use app\Urls\BaseClasses\HomegameYearUrlModel;
	use entities\Homegame;

	class CashgameListingUrlModel extends HomegameYearUrlModel{

		public function __construct(Homegame $homegame, $year){
			parent::__construct(RouteFormats::cashgameListing, RouteFormats::cashgameListingWithYear, $homegame, $year);
		}

	}

}

File: Web/Models/UrlModels/CashgameListingUrlModel.cs
using Core.Classes;
using Web.Routing;

namespace Web.Models.UrlModels{

	public class CashgameListingUrlModel : HomegameYearUrlModel{

		public CashgameListingUrlModel(Homegame homegame, int? year)
            : base(RouteFormats.CashgameListing, RouteFormats.CashgameListingWithYear, homegame, year)
        {
		}

	}

}